tests/playwright/resources/create-extension.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+extensionFolder=bootcextensionlocal
+
+yarn
+yarn build
+mkdir -p $extensionFolder
+cp -r packages/backend/dist/ $extensionFolder/dist
+cp packages/backend/package.json $extensionFolder/
+cp -r packages/backend/media/ $extensionFolder/media
+cp LICENSE $extensionFolder/
+cp packages/backend/icon-dark.png $extensionFolder/
+cp packages/backend/icon-light.png $extensionFolder/
+cp packages/backend/bootable.woff2 $extensionFolder/
+cp README.md $extensionFolder/
+
+# we need to move the folder with extension content into right podman desktop home folder
+mkdir -p tests/playwright/tests/output/bootc-tests-pd/plugins
+cp -r $extensionFolder tests/playwright/tests/output/bootc-tests-pd/plugins
```

Comment:
it can really be rewritten to a simpler two lines and placed directly into workflow.
```
mkdir -p tests/playwright/tests/output/bootc-tests-pd/plugins/bootcextensionlocal
cd packages/backend && cp -r dist/ package.json media/ LICENSE icon*.png bootable.woff2 README.md ../../tests/playwright/tests/output/bootc-tests-pd/plugins/bootcextensionlocal
```